lib/inspec/resources/service.rb

```diff
@@ -157,7 +157,12 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
       when "mac_os_x", "darwin"
         LaunchCtl.new(inspec, service_ctl)
       when "freebsd"
-        BSDInit.new(inspec, service_ctl)
+        version = os[:release].to_f
+        if version < 10
```

Comment:
Is there a reason to support FreeBSD 9 and earlier at this point given how terribly EOL those releases are?

Reply:
Well, that's really a question for you folks :) Wikipedia says that 10 came out in January 2014, and 9 was no longer supported since December 2016, so I think a good argument can be made that supporting 9 and earlier is no longer needed. However, right now the Inspec documentation says 9 is supported, so I made sure to maintain compatibility with it. Happy to go with whatever you think is best!